WooCommerce/Classes/View Modifiers/View+NoticesModifier.swift

```diff
@@ -36,35 +36,97 @@ struct NoticeModifier: ViewModifier {
         content
             .overlay(
                 buildNoticeStack()
+                    .padding()
                     .animation(.easeInOut, value: notice)
             )
     }
 
+    enum Constants {
```

Comment:
nit: Constants are usually kept private and after the main code

Reply:
Good point!